src/pages/library/DataBrowser.js

```diff
@@ -264,7 +264,7 @@ const Browser = () => {
       }, ['OK'])
     ]),
     h(SearchAndFilterComponent, {
-      fullList: dataCatalog, sidebarSections: extractCatalogFilters(dataCatalog),
+      fullList: dataCatalog, sidebarSections: _.map(filter => ({ ...filter, hideEmpty: true }), extractCatalogFilters(dataCatalog)),
```

Comment:
Consider:
```suggestion
      fullList: dataCatalog, sidebarSections: _.map(_.set('hideEmpty', true), extractCatalogFilters(dataCatalog)),
```

Reply:
Yes, that seems like a better way. We may end up deleting this but it's good to know for next time.